### PR TITLE
Update discord invite

### PR DESCRIPTION
**Pull request by neilZon**
# Description
Update discord links to use discord URL constant

### CODE REVIEW

**Comment by srinjoyc** [comment]
I think there's on in the footer and sidebar as well? Did we get those?

File: app/component-data/NavigationBar/joinFlowTabData.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import EcosystemIcon from "~/ui/design-system/images/content/ecosystem"
 import AmbassadorsIcon from "~/ui/design-system/images/nav/ambassadors"
 import SocialsIcon from "~/ui/design-system/images/nav/socials"
 import { Section } from "~/ui/design-system/src/lib/Components/NavigationBar/types"
+import { DISCORD_URL } from "~/ui/design-system/src/lib/constants"
```
Comment (10thfloor) [inline app/component-data/NavigationBar/joinFlowTabData.tsx:5]: Constants that are part of the site should not be defined in the design system.
Move this to `/app/constants/index.ts`

**Review by 10thfloor — CHANGES_REQUESTED** [review]

**Comment by srinjoyc** [comment]
Yep that's the one

**Comment by neilZon** [comment]
@srinjoyc in that case it should be covered by the PR as well!

**Review by 10thfloor — APPROVED** [review]

File: app/component-data/NavigationBar/joinFlowTabData.tsx (hunk 1 above)
Comment (10thfloor) [reply app/component-data/NavigationBar/joinFlowTabData.tsx]: Ty!

**Pull request merged**